Prototope/Layer.swift

```diff
@@ -253,13 +258,23 @@ public class Layer: Equatable, Hashable {
 	}
 
 	/** The scaling factor of the layer along the x dimension. Defaults to 1. */
-	public var scaleX: Double = 1 {
-		didSet { updateTransform() }
+	public var scaleX: Double {
+        get {
+            return layer.valueForKeyPath("transform.scale.x") as Double
```

Comment:
I wonder if getting these values from the presentation layer would be more what we want? :)